extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx

```diff
@@ -336,7 +341,9 @@ function TrackedMeasurementsContextProvider(
         // Magic string
         // load function added by our sopClassHandler module
         if (
-          displaySet.SOPClassHandlerId === SR_SOPCLASSHANDLERID &&
+          [SR_SOP_CLASS_HANDLER_ID, COMPREHENSIVE_3D_SR_SOP_CLASS_HANDLER_ID].includes(
```

Comment:
If you are goign to have this code twice it is worth extracting to a function up on line 28, and then just call the function to test the handlers.  However, the PREFERRED approach is to modify the display set extra attributes and to test:
displaySet.isOhifMeasurements  since that way you aren't embedding knowledge about the names of sop class handler modules inside the tracked measurements.